tendrl/commons/jobs/__init__.py

```diff
@@ -51,9 +51,7 @@ def _run(self):
 
                         # Job routing
                         if raw_job.get("payload", {}).get("tags", []):
-                            NS.node_context = NS.node_context.load()
-                            tags = json.loads(NS.node_context.tags)
-                            if set(tags).isdisjoint(
```

Comment:
write of NodeContext is exclusive only to
node-agent. In such a case, if integrations of tendrl like node-monitoring which has no rights to save
node-context to etcd can't execute a job. So, since the node_context containing tags is in-memory,
instead of loading the NodeContext from etcd, I thought the in-memory node-context can be used.